packages/Accordion/package.json

```diff
@@ -1,41 +1,29 @@
 {
   "name": "@blaze-react/accordion",
   "version": "0.6.0",
-  "publishConfig": {
-    "access": "public"
-  },
-  "description": "",
-  "main": "./lib/index.js",
+  "description": "Accordion component is a list of headers that hide or reveal additional content when selected",
+  "main": "./src/index.ts",
   "scripts": {
-    "build": "../../node_modules/.bin/rimraf ./lib && ../../node_modules/.bin/tsc --jsx react --esModuleInterop ./src/*.tsx ./src/**/*.tsx --outDir lib ",
+    "build": "NODE_ENV=production webpack --config webpack.config.js",
     "build-storybook": "build-storybook -c .storybook -o docs-build",
-    "lint": "../../node_modules/.bin/tslint -c ../../tslint.json src/**/*.tsx",
+    "lint": "eslint '*/**/*.{ts,tsx}' --ignore-path ../../.eslintignore --quiet --fix",
```

Comment:
The root folder have a husky hook and lint-staged rules to lint, test and build before being able to push. I can add same to individual packages though. I can create a new task to do so. What do you think?

Reply:
I'm maybe misunderstanding the workflow so happy to continue if everyone thinks its ok but my concern is that with `--fix` being the default it means someone could commit some code without running precommit using `-n` or merge something in git and it not get formatted properly and it wouldn't get picked up in ci because it would get fixed as part of the job  but not committed?

Reply:
I see. No problem removing `--fix`.